project/src/Behaviors/GSTArrive.gd

```diff
@@ -4,12 +4,17 @@ extends GSTSteeringBehavior
 # The calculation will attempt to arrive with zero remaining velocity.
 
 
+# The target whose location the agent will be steered to arrive at
 var target: GSTAgentLocation
+# The distance from the target for the agent to be considered successfully arrived
 var arrival_tolerance: float
+# The distance from the target for the agent to begin slowing down
 var deceleration_radius: float
+# A constant that represents the time it takes to change acceleration
```

Comment:
Constants should be constants, i.e.
```gdscript
const TIME_TO_REACH := 0.1
```
If it's meant as a setting, then I'd write something like: "Delay in seconds in the change of acceleration of the agent."
Note, I'm not sure what it's meant to do here.